spec/factories/tasks.rb

```diff
@@ -1,5 +1,9 @@
 FactoryBot.define do
   factory :task do
-    
+    sequence(:title, "title_1")
+    content {'サンプル'}
```

Comment:
スペースで開けた方が見やすいかと思います。
```suggestion
    content { 'サンプル' }
```

Reply:
スペースを追加しました。

Reply:
2bd5617 スペースを追加しました